src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &--4 {
+    width: 96px;
+    margin: 0 17px 0 0;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__star:nth-child(n + 5) {
+    background-image: url(../images/star.svg);
+  }
+}
```

Comment:
reuse the whole block from the Stars task to make it possible to set any amount of active stars